packages/client/src/components/SafeSettings/components/Assets/AssetTableView.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useClipboard } from "hooks";
+import { formatAddress, getFlowscanUrlForContract } from "utils";
+import { Copy } from "components/Svg";
+
+const AssetTableView = ({ assets, onRemoveClick }) => {
+    const clipboard = useClipboard();
+
+    return (
+        <div className="border-light rounded-sm py-3 table-border">
+            {assets.map(asset => {
+                const address = asset?.split(".")[1];
```

Comment:
Can we create a parseAsset util function? it takes the asset string as the argument and returns {address, name, ...otherStuff} that we we can just do const {address, name} = parseAsset(asset)

Reply:
Created a parseIdentifier function -> used it in all places, and removed getNFTMeta